unix/mkerrors.sh

```diff
@@ -11,7 +11,7 @@ unset LANG
 export LC_ALL=C
 export LC_CTYPE=C
 
-CC=${CC:-gcc}
+CC=${CC:-cc}
```

Comment:
Thanks for checking that, I was pretty sure it was but nice to know it really is.